### ISSUE TITLE
Issue launching server with Bountiful Baubles

### ISSUE THREAD

**Issue by AlexTheFixer** [SpongePowered/Sponge#3605]
### Affected Product(s)

SpongeForge

### Version

1.16.5-36.2.5-8.0.0-RC1055

### Operating System

Windows

### Java Version

1.16.5

### Plugins/Mods

```shell
For this instance the only conflicting mods will be listed:
Spongeforge
Bountiful Baubles
```

### Describe the bug

When launching the server it will list this line:
[09:10:02] [main/INFO] [STDERR/]: [java.lang.Throwable:printStackTrace:668]: Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionError: Critical injection failure: Variable modifier method onApplyDamageAmount(FLnet/minecraft/util/DamageSource;)F in bountifulbaubles.mixins.json:MixinPlayerEntity failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap bountifulbaubles-common-refmap.json
I reported a similar issue about 2 hours ago, however I assure you this is different.

**Comment by gabizou** [comment]
Should've been fixed a while back, please re-test with 1.16.5-36.2.5-8.0.0-RC1081.

**Comment by gabizou** [comment]
Not able to reproduce after testing with the build mentioned above.
